tests/unit/records/test_records_receivers.py

```diff
@@ -1145,3 +1146,33 @@ def test_populate_affiliation_suggest_does_nothing_if_record_is_not_institution(
     populate_affiliation_suggest(None, record)
 
     assert 'affiliation_suggest' not in record
+
+
+def test_populate_author_count():
+    schema = load_schema('hep')
+    schema_schema = schema['properties']['$schema']
```

Comment:
This thing I did was a terrible idea... we can avoid adding assertions for this.

Reply:
On the other hand, adding a test for the different `$schema` would be great.